### ISSUE TITLE
Autocorrect removes TODO warnings

### ISSUE THREAD

**Issue by erick-outware** [realm/SwiftLint#1959]
I am using autocorrect in a project, but the team used TODO to mark places in the code where we will have to come back to.

Before autocorrect I would see all the warning related to the TODOs statements, but since it's been on the warnings no longer appear.

I could not find a solution to it.
Is there a way to get the warnings back?

**Comment by marcelofabri** [comment]
Please use the provided issue template so we can have more information to help you.

**Comment by erick-outware** [comment]
Thanks Marcelo, here it is:

### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

##### Complete output when running SwiftLint, including the stack trace and command used

```bash
if which swiftlint >/dev/null; then
  swiftlint autocorrect
else
  echo "warning: SwiftLint not installed, download from https://github.com/realm/SwiftLint"
fi
```

### Environment

* SwiftLint version 0.22.0
* Installation method used: Homebrew
* Paste your configuration file:

```yml
disabled_rules:
  - closing_brace
  - opening_brace
  - variable_name
  - type_name
  - trailing_whitespace
  - force_cast
  - cyclomatic_complexity
  - line_length
  - force_try
  - function_body_length

included:
  - Source
  - Specs
excluded:
  - Carthage

line_length:
  warning: 135

type_body_length:
  warning: 300

file_length:
  warning: 500
  error: 1200
```

* Are you using [nested configurations](https://github.com/realm/SwiftLint#nested-configurations)?
No

* Which Xcode version are you using (check `xcode-select -p`)?
Xcode 9.1

* Do you have a sample that shows the issue? 
No

Our Swift code is marked with TODO notes like the one below:
```swift
// TODO: Add feature 321 here
```

All the TODO marks generate warnings during build, which is nice. It is a way of reminding us we need to go back and implement that feature/code.

But since I turned on `autocorrect` we no longer see the TODO warnings.
I am trying to get them back whilst still using autocorrect.

**Comment by marcelofabri** [comment]
`swiftlint autocorrect` doesn't report non-correctable rules, as [todo](https://github.com/realm/SwiftLint/blob/master/Rules.md#todo). 

You should run `swiftlint autocorrect` first and then `swiftlint lint` to get all violations.

**Comment by erick-outware** [comment]
Nice, thank you!
